Assets/Scripts/PlayerController/PlayerAnimation.cs

```diff
@@ -49,4 +53,31 @@ void Update()
                 m_Animator.speed = 1;
         }
     }
+
+    public void MakeCharacterFall() 
+    {
+        StartCoroutine("FallAction");
```

Comment:
Don't forget to check if the coroutine is already running first. If the coroutine is already running, we want to stop that coroutine first. Perhaps cache a boolean that you enable and disable in the coroutine? `IsCoroutineRunning` for example.

Reply:
Hmmmmm True that too